src/app/pages/website/blog/blog.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from "@angular/router";
+import {BlogComponent} from "./blog.component";
```

Comment:
Nit/: Please follow the formatting guidelines. Spaces around the import and single-quotes only

Reply:
fixed